pkg/cli/update.go

```diff
@@ -31,6 +32,11 @@ func Update(rack sdk.Interface, c *stdcli.Context) error {
 		target = v
 	}
 
+	if (target == current) {
+		c.Writef("No update to be performed... ")
```

Comment:
Let's be even more explicit, print `Already on the latest version <release>NUM</release>` and return nil instead of OK() 

(the ... OK pattern is for an action that needs time to complete and can fail with ERROR instead of OK.)